rest/route/github.go

```diff
@@ -222,23 +224,58 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 			}
 		}
 
-	// This case merely logs. EVG-19964 will add logic to create a version from the GitHub merge queue.
 	case *github.MergeGroupEvent:
-		grip.Debug(message.Fields{
+		var msg string
+		if disableMergeGroup {
+			msg = "merge group received, skipping"
+		} else {
+			msg = "merge group received, attempting to queue"
+		}
+		grip.Info(message.Fields{
 			"source":   "GitHub hook",
 			"msg_id":   gh.msgID,
 			"event":    gh.eventType,
 			"org":      event.GetOrg(),
 			"repo":     event.GetRepo(),
 			"base_sha": event.GetMergeGroup().GetBaseSHA(),
 			"head_sha": event.GetMergeGroup().GetHeadSHA(),
+			"message":  msg,
 		})
 
+		// This will be removed when EVG-19964 is ready.
+		if disableMergeGroup {
```

Comment:
will `disableMergeGroup` in general be removed? If yes, should we maybe move this comment up to where disableMergeGroup is set?

Reply:
Yes. It's a feature flag https://martinfowler.com/articles/feature-toggles.html. Copied the comment there.

Reply:
Makes sense. We should probably set it up like so: https://github.com/evergreen-ci/evergreen/commit/86a50a3828e2d30a165609da52c3fc403227b68b

Reply:
I don't think there's a need for that. For a feature like, say, a new way of doing host creation, flipping the feature flag has an instant effect on the entire system. In this case though, flipping the feature flag should change nothing at all, since it will be off for all projects. And then we can slowly roll it out for individual projects. I'm including this flag here just to be extra cautious during development.

Reply:
I guess that makes sense since we're expecting a short development cycle for this project.